contrib/bsp/src/mill/contrib/bsp/MillBuildServer.scala

```diff
@@ -64,13 +64,13 @@ class MillBuildServer(evaluator: Evaluator, bspVersion: String, serverVersion: S
     handleExceptions {
       val targets = getTargets(getModules(evaluator), evaluator)
 
-      // Remove the sortBy, reverse and distinctBy when https://youtrack.jetbrains.com/issue/SCL-17551 is resolved
-      new WorkspaceBuildTargetsResult(targets.sortBy(_.getId.getUri).reverse.distinctBy(_.getBaseDirectory).asJava)
```

Comment:
Removed the workaround since we are about to publish the fix in Idea https://github.com/JetBrains/intellij-scala/pull/534